model/cookies.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+import { insertValueAtIndex, swapItemPosition } from '~/model/arrayWrapper';
+
+export const removeCookieById = ({ cookies, cookieName, id }) => {
+    const cookiesData = cookies.get(cookieName);
+    const parsedData = cookiesData.split(',');
```

Comment:
you don't need this : `if (!cookiesData) throw new Error('Cookies are not set');`. there may not be a cookie with a specific name.